ppr-ui/src/components/unitNotes/UnitNoteReview.vue

```diff
@@ -1,23 +1,322 @@
+            </v-col>
+            <v-col cols="9">
+              {{ unitNoteType.header }}
+            </v-col>
+          </v-row>
+          <v-row no-gutters>
+            <v-col cols="3">
+              <h3>Document ID</h3>
+            </v-col>
+            <v-col cols="9">
+              {{ getMhrUnitNote.documentId }}
+            </v-col>
+          </v-row>
+          <v-divider class="my-3 mx-0" />
+          <v-row no-gutters>
+            <v-col cols="3">
+              <h3>Remarks</h3>
+            </v-col>
+            <v-col cols="9">
+              <p>{{ getMhrUnitNote.remarks }}</p>
+            </v-col>
+          </v-row>
+          <v-divider class="my-3 mx-0" />
+          <div class="px">
+            <h3>Person Giving Notice</h3>
+
+            <v-simple-table v-if="getMhrUnitNote" class="giving-notice-party-table" data-test-id="account-info-table">
+              <template v-slot:default>
+                <thead>
+                  <tr>
+                    <th class="px-0">
+                      Name
+                    </th>
+                    <th class="px-0">
+                      Mailing Address
+                    </th>
+                    <th class="px-0">
+                      Email Address
+                    </th>
+                    <th class="px-0">
+                      Phone Number
+                    </th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td class="person-name">
+                      <v-icon class="mt-n2">
+                        {{ getMhrUnitNote.businessName ? 'mdi-domain' : 'mdi-account' }}
+                      </v-icon>
+                      <span class="font-weight-bold">
+                        {{ givingNoticeParty.personName.first }}
+                        {{ givingNoticeParty.personName.middle }}
+                        {{ givingNoticeParty.personName.last }}
+                      </span>
+                    </td>
+                    <td class="">
+                      <base-address
+                        :editing="false"
+                        :schema="PartyAddressSchema"
+                        :value="givingNoticeParty.address"
+                      />
+                    </td>
+                    <td class="">
+                      {{ givingNoticeParty.emailAddress }}
+                    </td>
+                    <td class="">
+                      {{ toDisplayPhone(givingNoticeParty.phoneNumber) }}
+                      <span v-if="givingNoticeParty.phoneExtension">
+                        Ext {{ givingNoticeParty.phoneExtension }}
+                      </span>
+                    </td>
+                  </tr>
+                </tbody>
+              </template>
+            </v-simple-table>
+          </div>
+        </section>
+      </div>
+    </v-card>
+
+    <section id="mhr-unit-note-person-giving-notice" class="mt-15">
```

Comment:
I'm not sure `mt-15` is a valid helper class via vuetify, is it providing a margin here?

Reply:
Scratch that, just looked it up. Goes up to 16 now! Resolving.

Reply:
Yea, it gave a margin of 60px. I can use our own class as well.